tracer/src/Datadog.Trace.Tools.Runner/Gac/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿// <copyright file="AssemblyInfo.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Runtime.InteropServices;
+// ReSharper disable ConvertToAutoProperty
+
+namespace Datadog.Trace.Tools.Runner.Gac;
+
+// Code based on: https://github.com/dotnet/pinvoke/tree/main/src/Fusion
```

Comment:
Do we need to add Fusion to this file?
https://github.com/DataDog/dd-trace-dotnet/blob/master/LICENSE-3rdparty.csv

Reply:
Cool. Just checking!